### ISSUE TITLE
Split day clolumn to enable concurrent events

### ISSUE THREAD

**Issue by atticus-sullivan** [nlschn/semesterplanner#4]
Hi,
I started using your package for planning the semester. When planning at some points events take place at the same time (sometimes this resolves during planning, sometimes this works with e.g. lecture recordings).

Is there any way to be able to specify that a box representing a lecture should only span half the column for the day? (I thought using tikz this should be easy, but as far as I can see at this place you're using the `picture`-env (with which I'm not that familiar))

E.g. (despite I think it's pretty obvious)
```tex
\documentclass[a4paper, 12pt]{scrartcl}
\usepackage{semesterplanner}
\begin{document}
\begin{timetable}
    \seminar{sem}{E}{\tba}{T}{14:00-16:00}{\phigh}
    \lecture{lec}{E}{\tba}{T}{14:15-15:45}{\phigh}
    % in this case one can still see both events but in other occurrences
    % one might not be able to determine e.g. the endtime
\end{timetable}
\end{document}
```

**Comment by nlschn** [comment]
My fix for this was to split the two events in the timetable, i.e. set one from 14:00 to 15:00 and the other one from 15:00 to 16:00. This way, both events appear in the same slot (but beneath each other). 
As I am using a dedicated package for these timetables, I would have to look into this a bit closer or write my own timetable.

**Comment by atticus-sullivan** [comment]
Hm ok. I think splitting the events isn't such a good idea, since then one might think they are not concurrent.

Over the day I experimented a bit, writing an own timetable. Because I'm not that experienced with (plain) tex I decided to write in Lua. The drawback of course is that the script only can be run with **lua(la)tex**. I'm not completely finished yet (some options I'd like to introduce, the docs are missing yet and learn about `.sty` files, since up to now this is simply a `.tex` file I'm including). But if you're interested just let me know.

In the long run (will take time especially with the `.sty` thing) I'm thinking of possibly publishing (for now only on github/gitlab, since I have no idea how publishing on e.g. ctan works and I'd first need to check if the code quality is good enough).
But I'm not sure I can simply do that, since I implemented some sort of generic timetable (only drawing the grid and positioning the tikz-nodes) and resorted then back to your code in terms of symboldrawing, the colors you defined and I've "stolen" the formatting of the entries.
So I'd be feeling much better if you'd give permission after viewing (or decline).
